### ISSUE TITLE
Updated "Python visualization landscape" Diagram for the Overviews page.

### ISSUE THREAD

**Issue by dharhas** [pyviz/pyviz.org#124]
Hey Folks,

We are getting ready to publish a blog post on visualization and we wanted to include an updated "Python visualization landscape" diagram. 

I've seen the WIP on #122 and that looks like a great way of thinking about things but for our purposes, we like the structure of the existing diagram since it sends the message that the libraries target different runtimes so to speak: javascript, matplotlib, opengl, desktop etc.

This is really short notice (we are hoping to publish our blog post early next week), but I wanted to see if I could get any feedback from folks on:

- what packages are now outdated and should be dropped from the viz
- what new packages should be added and where should they be connected (i.e hvplot, napari etc)

We are going for a minor update that keep the same main structure of the existing diagram.

Any feedback we can get this weekend is appreciated and we will probably make a static image for the blog and then continue to work here to make an actual PR to improve the pyviz version.

@kcpevey @philippjfr @jbednar @pierrotsmnrd

**Comment by dharhas** [comment]
Some proposed packages to add:

- hvplot
- napari
- geoviews

**Comment by dharhas** [comment]
I guess we could cross reference with: https://pyviz.org/tools.html#dormant-projects

**Comment by jbednar** [comment]
Right; I don't think any dormant projects should be in the diagram.

**Comment by jbednar** [comment]
Closing this for now, but still happy to see updated diagrams once people are happy with them enough to publish them!
